### PR TITLE
Fix codespell

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: .codespellrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [codespell]
 skip = ./testing/**/*,./.git/**/*,./application/cxxopts.hpp,./cmake/NSIS.template.in
-ignore-words-list=nnumber,unknwn,dota
+ignore-words-list=nnumber,unknwn,dota,superseed
```
Comment (Meakk) [inline .codespellrc:3]: Why do you want to whitelist it instead of fixing errors?

Comment (mwestphal) [reply .codespellrc:3]: superseed is a word

Comment (Meakk) [reply .codespellrc:3]: It's not ^^

Comment (mwestphal) [reply .codespellrc]: turns out its not

**Review by Meakk — APPROVED** [review]

**Pull request merged**